### DISCUSSION TITLE
tuiDropdownHover не работает с tui-accordion

### DISCUSSION THREAD

**Discussion by omqmo** [taiga-family/taiga-ui#5523] [Q&A] [answered]
Самая обычная менюшка - dropdown с аккордином внутри. Не работает. По наведению курсора dropdown открывается и сразу же закрывается. Наверняка создатели сразу же поняли в чем дело, что избавит меня от утомительно процесса воссоздания проблемы. И да, код самый обычный, фактически копипаста из доков, так что сетовать на супер сложную ,криво сделанную лично мной, интерактивную логику вложенную в dropdown - не стоит. Все бонально примитивно и без аккордиона работает.  Это можно, как-то исправить?

**Comment by waterplea** [comment, marked as answer]
Это вот этот баг, постараемся поскорее его пофиксить:
https://github.com/taiga-family/taiga-ui/issues/5395
